modules/node/main.tf

```diff
@@ -8,5 +8,9 @@ resource "equinix_metal_device" "node" {
   count            = var.node_count
   billing_cycle    = "hourly"
   project_id       = var.project_id
+  ip_address {
```

Comment:
This should be removed. There was a starting thought that all nodes could use the EM Private IPs, but DHCP management needs, configuring a gateway, meant that VRF and DHCP services running on the bastion would be a better fit.